lib/chef-dk/helpers.rb

```diff
@@ -103,8 +112,17 @@ def usr_bin_path(command)
     # somewhere else that we can append to the end of the path.
     # This is only a temporary solution - see https://github.com/chef/chef-dk/issues/854
     # for a better proposed solution.
+    # Note that we are not including git in the path of the habitat packages,
+    # as we begin to move away from embedding git.
     def git_bin_dir
-      @git_bin_dir ||= File.expand_path(File.join(omnibus_root, "gitbin"))
+      @git_bin_dir ||=
+        begin
+          if habitat_install?
+            ""
```

Comment:
so what you should do here is continue to ship git with the habitat package, but you should return the path to it here, but then ensure that path does NOT wind up getting jammed into the user's PATH/Path.  all that should go into the user's PATH is ruby and the binstubs.